CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## [2.5.0] - 2022/09/08
```

Comment:
Maybe a major version change might be appropriate since `flutter_web_auth_2` does not support Flutter < 3 anymore? @okrad